### PR TITLE
Fix WTM_top_coin file not found on first run

### PR DESCRIPTION
**Pull request by papampi**
Prevent first run error by creating WTM_top_coin if not exists
issue https://github.com/papampi/nvOC_by_fullzero_Community_Release/issues/138

### CODE REVIEW

**Comment by papampi** [comment]
https://github.com/papampi/nvOC_by_fullzero_Community_Release/issues/138

File: 8wtm_auto_switch [hunk 1]
```diff
@@ -78,9 +80,17 @@ if [[ -e "$WTM_HISTORY" ]] ; then
...
 FIRST_COIN="$(echo "$WTM_AUTO_SWITCH_COINS" | awk -F'[_;]' '{print $1}')"
+
+# Prevent first run error on no top coin
+if [[ ! -e ${NVOC}/WTM_top_coin ]]
+then
+  echo "$FIRST_COIN:100" > ${NVOC}/WTM_top_coin
+fi
+
 TOP_COIN=$(sed 's/:[0-9]*//' < ${NVOC}/WTM_top_coin)
+
```
Comment (LuKePicci) [inline 8wtm_auto_switch]: Avoid immediate read after write by setting TOP_COIN=$FIRST_COIN in the above if block and move the read from file command at line 91 in its else branch.

**Review by LuKePicci — CHANGES_REQUESTED** [review]

File: 8wtm_auto_switch (hunk 1 above)
Comment (papampi) [reply 8wtm_auto_switch]: Better now?

Comment (LuKePicci) [reply 8wtm_auto_switch]: Have you pushed the new commit?

Comment (papampi) [reply 8wtm_auto_switch]: Yup, but not showing up 
No idea why

Comment (papampi) [reply 8wtm_auto_switch]: Showing in branch, but not in PR
https://github.com/papampi/nvOC_by_fullzero_Community_Release/blob/papampi-WTM_top_coin-patch/8wtm_auto_switch

Comment (LuKePicci) [reply 8wtm_auto_switch]: I see, seems that server are overloaded a.t.m.

**Review by LuKePicci — APPROVED** [review]

**Pull request merged**